rpcserver.go

```diff
@@ -4120,6 +4137,40 @@ func handleSetGenerate(s *rpcServer, cmd interface{}, closeChan <-chan struct{})
 	return nil, nil
 }
 
+// handleGetMempoolFee implements the getmempoolfee command. We will return
```

Comment:
I think this comment is just copied from the func above, should update for this function.